cypress/elements/BasePage.js

```diff
@@ -0,0 +1,33 @@
+import { NEW_ITEMS } from './../helpers/toDoArray'
+
+export default class BasePage {
+
+  getItemsFromLocalStorage () {
```

Comment:
I'd prefer to see LocalStorage actions into separate class. It doesn't related to page objects directly.